src/super_gradients/recipes/imagenet_resnet50.yaml

```diff
@@ -6,10 +6,10 @@
 #  Log and tensorboard at s3://deci-pretrained-models/ResNet50_ImageNet/average_model.pth
 
 # Instructions:
-# running from the command line, set the PYTHONPATH environment variable: (Replace "YOUR_LOCAL_PATH" with the path to the downloaded repo):
-#   export PYTHONPATH="YOUR_LOCAL_PATH"/super_gradients/
-# Then:
-#   python train_from_recipe_example/train_from_recipe.py --config-name=imagenet_resnet50
+#   0. Make sure that the data is stored in dataset_params.dataset_dir or add "dataset_params.data_dir=<PATH-TO-DATASET>" at the end of the command below (feel free to check ReadMe)
```

Comment:
Don't think this file was supposed to be in this PR...

Reply:
This was part of previous PR that is now merged. Now it is removed fro this PR.